chef/cookbooks/network/recipes/default.rb

```diff
@@ -393,6 +393,20 @@ def get_datapath_id_for_ovsbridge(bridge)
     nic.tx_offloading = node["network"]["enable_tx_offloading"] || true
   end
 
+  # if the nic is a vlan and the physical parent is not used, also set
+  # the mtu from the vlan nic for the parent
+  if nic.kind_of?(Nic::Vlan)
+    unless ifs[nic.parent].key? "mtu"
```

Comment:
Good point. But it would be nice to validate things like this **before** we apply the config. If a user sets an MTU of 1500 for a physical interface and sets a higher MTU for the vlan network, the configuration is wrong.
Are we doing already some network validation somewhere?